carbon_black_cloud/help.md

```diff
@@ -15,8 +15,6 @@ The VMware Carbon Black Cloud is a cloud-native endpoint protection platform (EP
 # Documentation
 ## Setup
 
-For information on how to get the API Key, API Secret, and your base URL please see the [Carbon Black Authentication](https://developer.carbonblack.com/reference/carbon-black-cloud/authentication/) documentation.
```

Comment:
Did you mean to remove this? Looks like it's useful

Reply:
This is getting removed on regen